output_definition.json

```diff
@@ -1,61 +1,78 @@
 {
     "mirna_seq_pipeline.cutadapt": {
         "no3ad_untrimmed_fastq": {
-            "path": "cutadapt/rep${i+1}/${basename}"
+            "path": "cutadapt/rep${i+1}/${basename}",
+            "table": "Cutadapt/Replicate ${i+1}/Fastq of reads with no 3' adapter"
         },
         "no5ad_untrimmed_fastq": {
-            "path": "cutadapt/rep${i+1}/${basename}"
+            "path": "cutadapt/rep${i+1}/${basename}",
+            "table": "Cutadapt/Replicate ${i+1}/Fastq of reads with no 5' adapter"
         },
         "too_short_fastq": {
-            "path": "cutadapt/rep${i+1}/${basename}"
+            "path": "cutadapt/rep${i+1}/${basename}",
+            "table": "Cutadapt/Replicate ${i+1}/Fastq of reads that are too short"
         },
         "trimmed_fastq": {
-            "path": "cutadapt/rep${i+1}/${basename}"
+            "path": "cutadapt/rep${i+1}/${basename}",
+            "table": "Cutadapt/Replicate ${i+1}/Trimmed fastq"
         }
     },
     "mirna_seq_pipeline.star": {
         "bam": {
-            "path": "star/rep${i+1}/${basename}"
+            "path": "star/rep${i+1}/${basename}",
+            "table": "Star/Replicate ${i+1}/Alignments bam"
         },
         "tsv": {
-            "path": "star/rep${i+1}/${basename}"
+            "path": "star/rep${i+1}/${basename}",
+            "table": "Star/Replicate ${i+1}/Quantifications tsv"
         },
         "plus_strand_all_wig": {
-            "path": "star/rep${i+1}/${basename}"
+            "path": "star/rep${i+1}/${basename}",
+            "table": "Star/Replicate ${i+1}/Plus strand wig of all reads"
         },
         "minus_strand_all_wig": {
-            "path": "star/rep${i+1}/${basename}"
+            "path": "star/rep${i+1}/${basename}",
+            "table": "Star/Replicate ${i+1}/Minus strand wig of all reads"
         },
         "plus_strand_unique_wig": {
-            "path": "star/rep${i+1}/${basename}"
+            "path": "star/rep${i+1}/${basename}",
+            "table": "Star/Replicate ${i+1}/Plus strand wig of uniquely mapped reads"
         },
         "minus_strand_unique_wig": {
-            "path": "star/rep${i+1}/${basename}"
+            "path": "star/rep${i+1}/${basename}",
+            "table": "Star/Replicate ${i+1}/Minus strand wig of uniquely mapped reads"
         },
         "star_qc_json": {
-            "path": "star/rep${i+1}/${basename}"
+            "path": "star/rep${i+1}/${basename}",
+            "table": "Star/Replicate ${i+1}/Quality control json"
         },
         "star_qc_log": {
-            "path": "star/rep${i+1}/${basename}"
+            "path": "star/rep${i+1}/${basename}",
+            "table": "Star/Replicate ${i+1}/Python log"
```

Comment:
NIT:  Maybe STAR log?

Reply:
This one is actual python log from the script that produces the quality control json, so the naming is accurate as it is.